customer-endpoint/src/main/kotlin/org/ostelco/prime/customer/endpoint/resources/ProductsResource.kt

```diff
@@ -54,4 +55,31 @@ class ProductsResource(private val dao: SubscriberDAO) {
                         saveCard = saveCard ?: false)
                         .responseBuilder(Response.Status.CREATED)
             }.build()
+
+    @POST
+    @Path("{sku}/renew")
+    @Produces(MediaType.APPLICATION_JSON)
+    fun renewSubscription(@Auth token: AccessTokenPrincipal?,
```

Comment:
Just the word `Subscription` means mobile Subscription (with MSISDN as ID).  Better to call this `PaymentSubscription` or `PlanSubscription` to distinguish it from "Mobile" Subscription.